proto/api/v1/api.proto

```diff
@@ -1,12 +1,13 @@
 syntax = "proto3";
 
 package api.v1;
-option go_package = "github.com/conduitio/conduit/proto/api/v1;api";
 
+import "opencdc/v1/opencdc.proto";
 import "google/api/field_behavior.proto";
 import "google/api/annotations.proto";
 import "protoc-gen-openapiv2/options/annotations.proto";
 import "google/protobuf/timestamp.proto";
+import "google/protobuf/descriptor.proto";
```

Comment:
Do we need this import? It's strange, because the linter doesn't complain that it's there, but it also doesn't complain if I remove it 🤔

Reply:
TBH, I have no idea how it got here.:) I can't remember adding it myself, but could have a copy-paste mistake. I'll remove it, thanks for noticing it! And yes, definitely interesting that the linter doesn't complain either way.:S